src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -387,8 +386,15 @@ internal SeriesSummary GetSummary()
                             continue;
                         }
 
-                        result.Min[i] = Math.Min(result.Min[i], _currentSegment.SegmentValues.Span[i].Min);
-                        result.Max[i] = Math.Max(result.Max[i], _currentSegment.SegmentValues.Span[i].Max);
+                        if (double.IsNaN(_currentSegment.SegmentValues.Span[i].Min) == false)
+                        {
+                            result.Min[i] = Math.Min(result.Min[i], _currentSegment.SegmentValues.Span[i].Min);
+                        }
+
+                        if (double.IsNaN(_currentSegment.SegmentValues.Span[i].Max) == false)
```

Comment:
Why did you add these?

Reply:
otherwise, if the segment has an item with missing values (NaN), then min and max will both be double.NaN